extensions/wikia/Recirculation/services/PopularPagesService.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+class PopularPagesService {
+	public function getPopularPages( int $limit, int $thumbWidth, int $thumbHeight ): array {
+		global $wgCityId, $wgContentNamespaces;
+
+		$articles = DataMartService::getTopArticlesByPageview(
```

Comment:
as far as I can see this method queries DB directly. Do we want to cache it somehow? Or use WikiService class maybe? It caches data for 24h

Reply:
This method uses a cache internally for 1 day

Reply:
I looked at wrong method :p